projects/ngu-carousel/src/lib/ngu-carousel/ngu-carousel.ts

```diff
@@ -58,11 +58,25 @@ export class Touch {
 }
 
 export class Transfrom {
+  public xl?:number = 0;
   constructor(public xs = 0, public sm = 0, public md = 0, public lg = 0, public all = 0) {}
 }
```

Comment:
This looks a little strange since I didn't include the xl in the constructor but that was in order to ensure backwards compatibility